src/ui/mod.rs

```diff
@@ -0,0 +1,28 @@
+//! Collection of UI helpers.
+
+mod mouse_hover;
+mod widgets;
+
+use bevy::{color::palettes::tailwind, prelude::*};
+
+pub use widgets::*;
```

Comment:
IMO it would be better to `pub mod widgets` instead. But that's just my opinion, and can be discussed after this PR is merged.